Octokit/Clients/AuthorizationsClient.cs

```diff
@@ -36,7 +36,7 @@ public AuthorizationsClient(IApiConnection apiConnection) : base(apiConnection)
         /// <returns>A list of <see cref="Authorization"/>s.</returns>
         public Task<IReadOnlyList<Authorization>> GetAll()
         {
-            return ApiConnection.GetAll<Authorization>(ApiUrls.Authorizations(), null);
+            return ApiConnection.GetAll<Authorization>(ApiUrls.Authorizations(), (IDictionary<string,string>)null);
```

Comment:
This is pretty :hankey: but I guess managing the overloads of the classes like ```ApiConnection``` gets pretty tricky with yet another option, and no way for the compiler to know when you call ```GetAll(string, null)``` which of the 2 parameter overloads you intended.  Though it would be nice to see whether the overloads can be consolidated to avoid this (or is the intention that things like the accepts header of timeout would shift into this new ```ApiOptions``` parameter?  Even so I guess to maintain backwards compat, most of the overloads would have to stay (or can we just cop a breaking change, given something like implementing pagination support is pretty major, and you arent actually out of beta yet)?